pages/api/user-donations.js

```diff
@@ -0,0 +1,55 @@
+import Iron from '@hapi/iron'
+import CookieService from '../../lib/cookie'
+import { getUnix } from '../../utils/date.utils'
+
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY)
+
+const getCustomer = async ({ email }) => {
+  return stripe.customers.list({ email })
+}
+
+const getCustomerCharges = async (opts) => {
+  return stripe.charges.list(opts)
+}
+
+export default async (req, res) => {
+  const customerTransactions = { data: { transactions: [], user: {} } }
+  try {
+    const user = await Iron.unseal(CookieService.getAuthToken(req.cookies), process.env.ENCRYPTION_SECRET, Iron.defaults)
+    const customerResponse = await getCustomer({ email: user.email })
+
+    const customer = customerResponse.data[0]
+    if (customer) {
```

Comment:
here you can break early by doing 

`if (!customer) return` or something like that, then the rest of the method doesn't have to be as indented